recipes/uri-template/all/conanfile.py

```diff
@@ -0,0 +1,106 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class UriTemplateConan(ConanFile):
+    name = "uri-template"
```

Comment:
```suggestion
    name = "tinkoffcreditsystem-uri-template"
```
Same comment as done for #5505

https://github.com/search?q=uri%2Btemplate

Reply:
Renamed the package